packages/thumbprint-react/components/ServiceCard/test.tsx

```diff
@@ -2,14 +2,17 @@ import React from 'react';
 import { mount } from 'enzyme';
 import ServiceCard, { ServiceCardImage, ServiceCardTitle, ServiceCardDescription } from './index';
 
-jest.mock('react-intersection-observer', () => ({
-    InView: ({ children }) => children({ inView: true, ref: null }),
-    useInView: () => [() => {}, true],
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Jest doesn't have types for this stuff?

Reply:
it does, but here we are mocking out with dummy values that do not match the actual type signature, so it's best to ignore it.

Reply:
I've seen a bunch of people recommend avoiding `any` if at all possible. This seems like a case where it's maybe reasonable, since it's a mock. Would it be overkill to avoid using `any`?

Reply:
It's possible to avoid any here, but the resulting types are very hard to read. You're basically mocking out a real library with fake values so it's lots of function returning `void`, `null`, etc. Anyway, I did it, so you can take a look and see if you prefer this, or the `any`  stuff.